docs/admin/audit-logs.md

```diff
@@ -1,10 +1,11 @@
 # Audit Logs
 
-This is an enterprise feature that allows **Admins** and **Auditors** to monitor what is happening in their deployment.
+Audit Logs allows **Admins** and **Auditors** to monitor user operations in
+their deployment.
 
 ## Tracked Events
 
-This feature tracks **create, update and delete** events for the following resources:
+We tracks **create, update and delete** events for the following resources:
```

Comment:
There is a typo here. It should be "track".

Reply:
thx